Themes/default/webfonts/index.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// Try to handle it with the upper level index.php. (it should know what to do.)
+if (file_exists(dirname(__DIR__) . '/index.php')) {
+	include dirname(__DIR__) . '/index.php';
+} else {
+	exit;
+}
+
+?>
```

Comment:
```suggestion
}

```

We're no longer using closing PHP tags as of #8606.